inst/include/population_dynamics/population/population.hpp

```diff
@@ -118,20 +81,8 @@ struct Population : public fims_model_object::FIMSObject<Type> {
 
     // size all the vectors to length of n_ages
     n_fleets = fleets.size();
```

Comment:
same comment as fleet Initialize and Prepare.

Reply:
same response as fleet Initialize and Prepare.